Documents/public-api.md

```diff
@@ -34,3 +34,54 @@ This documenatation describes de public HMDA Platform HTTP API
      ]
    ```
 
+### Modified LAR
+
+* `/institutions/<institutionId>/filings/<period>/lar`
+
+   * `GET` - Returns the Modified LAR, in CSV format. The schema for the data is as follows:
+
+
+   id
```

Comment:
I think GitHub will format this list correctly if you use bullets or surround the whole thing in triple backtics. Currently, when I preview the file, it's making them all into one paragraph.